src/pages/patterns/login-pattern/index.mdx

```diff
@@ -59,104 +55,290 @@ The login page is presented to users in the following scenarios:
+#### Designing for multiple alternate logins
 
-### Split-screen layout
+As mentioned above, we prefer that the system distinguishes the path a user needs to take in the background rather than making them choose in the UI. However, with certain products, that’s not an option. In order to present multiple alternate logins to the user up front, designers must use fixed text inputs and fixed buttons so that the primary button can remain close to the input field.
 
-The split-screen page is an alternate design that can be used to include some marketing content or other visual treatment related to the product. The login portion of this layout uses the same design and behavior as the single-column layout but is confined to one part of the page.
+Avoid layouts that emphasize alternate logins over the preferred login path.
 
 <Row>
 <Column colLg={12}>
 
-![Login with visual imagery](/images/Login_with_graphic.png)
+![Example of a log in flow using multiple alternate logins](./images/7_Login_multiple_alts_1120.png)
 
 <Caption>
-  Example of a split-screen login page with product-related visual imagery
+  Keep the primary button closest to the text input when presenting multiple
+  alternate logins
 </Caption>
 
 </Column>
 </Row>
 
+<DoDontRow>
+<DoDont
+type="dont"
+caption="Do not put alternate login buttons between the username input and the primary button.">
+
+![Do not put alternate login buttons between the username input and the primary button](./images/8_dont_alt_logins_544.png)
+
+</DoDont>
+
+<DoDont
+type="dont"
+caption="Do not put alternate login buttons at the top of the login form.">
+
+![Do not put alternate login buttons at the top of the login form](./images/9_dont_alt_logins_544.png)
+
+</DoDont>
+</DoDontRow>
+
+### Position
+
+Carbon provides best practice advice on the login pattern but will leave more specific design guidance to the subsystem. For instance, decisions like where to position the login flow on a page (i.e. left, right or center), or whether to use fluid or fixed inputs, can be made at the subsystem level as long as the fields remain on the grid. Designers can also choose whether to incorporate brand-approved subsystem background textures, illustrations and/or marketing content. Vist the [IBM Brand Center](https://www.ibm.com/brand/) for specific guidance and approved assets relating to your subsystem.
+
+#### Centered layout
+
+Placing the fluid login form in the center of the screen creates a simple entry point for users. Without any distractions on the page, users can focus on their primary goal of signing in to the application or product. Through navigating to the login page, users have already shown intent to log in so additional content about the product isn’t necessary.
+
+This type of login is often paired with a solid color background or a subsystem approved background texture. Because the form is the focal point, complex illustrations are not appropriate in this situation.
```

Comment:
```suggestion
This type of login is often paired with a solid color background or a subsystem-approved background texture. Because the form is the focal point, complex illustrations are not appropriate in this situation.
```